ambry-replication/src/test/java/com.github.ambry.replication/ReplicationTest.java

```diff
@@ -1840,6 +1840,7 @@ void addMessage(PartitionId id, MessageInfo messageInfo, ByteBuffer buffer) {
                 new RemoteReplicaInfo(peerReplicaId, replicaId, store, new MockFindToken(0, 0), Long.MAX_VALUE,
                     SystemTime.getInstance(), new Port(peerReplicaId.getDataNodeId().getPort(), PortType.PLAINTEXT));
             remoteReplicaInfos.add(remoteReplicaInfo);
+            break;
```

Comment:
It's possible to have two replicas(belong to same partition) on same hosts. We can't assume there is only one, right?

Reply:
I made the change based on the assumption that no two replicas belonging to same partition would reside on same node. Since this is only a test and we may allow two replicas on single node in the future, so I decide to remove the break here.